src/components/WeekTime/index.ts

```diff
@@ -0,0 +1,4 @@
+import WeekTime from "./WeekTime";
+import WeekTimeSelector from "./WeekTimeSelector";
+
```

Comment:
https://github.com/voyagegroup/ingred-ui/pull/1461#issuecomment-1758943392 
ここかな

```suggestion
export type { WeekTimeProps } from "./WeekTime";
export type { WeekTimeSelectorProps } from "./WeekTimeSelector";
```